extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/user/ModeledAuthenticatedUser.java

```diff
@@ -79,6 +84,7 @@ public ModeledAuthenticatedUser(AuthenticatedUser authenticatedUser,
         super(authenticatedUser.getAuthenticationProvider(), authenticatedUser.getCredentials());
         this.modelAuthenticationProvider = modelAuthenticationProvider;
         this.user = user;
+        this.setAttributes(authenticatedUser.getAttributes());
```

Comment:
I think this properly should be:

    super.setAttributes(authenticatedUser.getAttributes());

since `setAttributes()` is defined in `RemoteAuthenticatedUser`.